src/styles.ts

```diff
@@ -0,0 +1,142 @@
+import styled, { createGlobalStyle } from "styled-components";
+import { HashLink } from "react-router-hash-link";
+import img from "./highlighter.png";
+
+export const GlobalStyle = createGlobalStyle`
+    body {
+        font-family: 'Josefin Sans', sans-serif;
+        font-weight: 300;
+        font-size: 18px;
+        };
+
+    h1 {
+        font-weight: 700;
+        font-size: 45px;
+        text-shadow: 4px 4px 5px #9ab0af;
+        align-self: center;
+        text-align: center;
+        margin-bottom: 5px;
+
+        @media screen and (max-width: 1668px) {
+          font-size: 40px;
+         }
+    };
+
+    h2 {
+        font-weight: 700;
+        font-size: 25px;
+
+        @media screen and (max-width: 1668px) {
+          font-size: 18px;
+         }
+    };
+
+    h3 {
+        font-weight: 400;
+        font-size: 18px;
+    };
+    
+    h4 {
+        font-weight: 400;
+        font-size: 20px;
+    }
+`;
+
+export const theme = {
```

Comment:
I didn't see any place where you used hardcoded value for color or fontFamily in the code - very nice 👏